README.md

```diff
@@ -2,18 +2,19 @@
 
 A package of resources for Hypothesis front-end applications.
 
-#### Requirements
+## Requirements
 
-- preact
-- browserify
+Your project must have Preact installed as a dependency.
 
-### Usage
+`yarn install --save preact`
```

Comment:
Can I suggest changing this to `npm install` for consistency with the command below.